src/OMSimulatorPython/system.py

```diff
@@ -202,6 +202,31 @@ def addSSV(self, cref: CRef, resource: str):
         raise ValueError(f"Component '{first}' not found in {self.name}")
       self.elements[first].addSSV(resource)
 
+  def split_cref(self, cref: CRef):
+    if cref.is_root():
+      return ("", str(cref))
+    else:
+      return (cref.first(), cref.last())
```

Comment:
This doesn't work with crefs that have more than twp parts, e.g. XX.YY.ZZ. YY would just be dropped.

Also, the method shoul dbe part of the CRef class.